### PR TITLE
contacts | update 2020

### PR DESCRIPTION
**Pull request by julbinb**

### CODE REVIEW

File: contact.rkt [hunk 1]
```diff
@@ -43,46 +43,59 @@
...
+                @div[class: "row"]{
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{PRL Social Virtually} @a[href: "mailto:mhyee@ccs.neu.edu"]{Ming-Ho Yee} @br{}
+                    PRL parties, tea time, bingewatching
+                  }
+
+                  @div[class: "col-xs-4 pad-contact"]{
+                    @h4{Twitter @br{} 
+                      @small{@linksym["@neu_prl" "https://twitter.com/neu_prl"]}} 
+                      @a[href: "mailto:mhyee@ccs.neu.edu"]{Ming-Ho Yee}
```
Comment (mhyee) [inline contact.rkt:83]: @aatxe is now the main Twitter maintainer. (I still have access as backup, but I don't actively maintain it now.)

Comment (julbinb) [reply contact.rkt:83]: Ah, sure. Somebody, fix this, please!

Comment (ulysses4ever) [reply contact.rkt:83]: Let me look into it: I think I could improve CSS a little bit on the way.

**Comment by ulysses4ever** [comment]
Updated version of this (based on comments) is in #265.

**Pull request closed without merging**